clients/fides-js/src/lib/tcf.ts

```diff
@@ -62,7 +56,7 @@ export const generateTcString = async ({
   // due to TCF library not yet supporting latest GVL (https://vendor-list.consensu.org/v3/vendor-list.json).
   // We'll need to update this with our own hosted GVL once the lib is updated
   // https://github.com/InteractiveAdvertisingBureau/iabtcf-es/pull/389
-  const tcModel = new TCModel(new GVL(gvlJson as VersionOrVendorList));
+  const tcModel = new TCModel(new GVL(experience.gvl));
```

Comment:
Can gvl ever be undefined in TCF mode? should we catch / log in the case of potentially hitting this state?

Reply:
yeah that's a good idea—I'll add something!

Reply:
added a try catch: https://github.com/ethyca/fides/pull/4159/commits/18a9c2c26a6965bfd99bbc8aed7329dd4075794e